src/index.js

```diff
@@ -1,51 +1,45 @@
 import './styles/style.css';
 
 const todoList = [
-    {
-    "index": "1",
-    "description": "Booking my flight tomorrow",
-    "completed": false
-},
-{
-    "index": "2",
-    "description": "Visiting my children school friday",
-    "completed": false
-},
-{
-    "index": "3",
-    "description": "Going for groceries shopping",
-    "completed": false
-},
-{
-    "index": "3",
-    "description": "Visiting the gym",
-    "completed": false
-}
+  {
+    index: '1',
+    description: 'Booking my flight tomorrow',
+    completed: false,
+  },
+  {
+    index: '2',
+    description: 'Visiting my children school friday',
+    completed: false,
+  },
+  {
+    index: '3',
+    description: 'Going for groceries shopping',
+    completed: false,
+  },
+  {
+    index: '3',
+    description: 'Visiting the gym',
+    completed: false,
+  },
 ];
 
-console.log(todoList);
-
 const showTodo = () => {
-    let listContainer = document.querySelector('.list-items');
-    let placeholder = '';
+  const listContainer = document.querySelector('.list-items');
+  let placeholder = '';
 
-    todoList.forEach((todo) => {
-        placeholder +=`
+  todoList.forEach((todo) => {
+    placeholder += `
         <li>
             <div>
                 <input type="checkbox" class="checkbox" name="checkbox">
                 <lable>${todo.description}</lable>
             </div>
             <i class="fas fa-ellipsis-v"></i>
         </li>
-        `;  
-    });
-
-    listContainer.innerHTML = placeholder;  
+        `;
+  });
```

Comment:
- [ ] Impressive work displaying the list of tasks. It would be better if you `sort` the list of tasks based on their index before displaying it on the UI as specified in the project requirement. This will help sort your task to display orderly without having to worry about ordering them in your array. 
 - Example: If you have a list of tasks in an array with an index such as 
`index: 10, index: 12, index: 5, index: 7` The sort method will sort the index like 
`eg: index: 5, index: 7, index: 10, index:12` 
See below for more details on project requirements:
<img width="996" alt="Screenshot 2023-06-07 at 15 15 43" src="https://github.com/Innocent259/Todo-List/assets/107192076/ded13da6-12fb-4d6d-b89d-764fc5f6c6b2">